packages/hooks/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
should prettier files be consolidated in the root of the monorepo too?

Reply:
Added to #33